### PR TITLE
[RFR] [Feature] add target to FileField

### PR DESCRIPTION
**Pull request by natrim**
sometimes you really need to open the attachments in new window

### CODE REVIEW

**Comment by fzaninotto** [comment]
Please add documentation, too

**Comment by natrim** [comment]
@fzaninotto tried .)

**Comment by natrim** [comment]
any other needed changes?

File: docs/Fields.md [hunk 1]
```diff
@@ -240,6 +240,12 @@ If the record actually contains an array of files in its property defined by the
 <FileField source="files" src="url" title="desc" />
 ```
 
+You can optionally set `target` prop to choose which window will the link try to open in.
```
Comment (djhi) [inline docs/Fields.md]: s/optionally set/optionally set the/

Comment (natrim) [reply docs/Fields.md]: changed and repushed

**Comment by djhi** [comment]
Thanks!

**Comment by Cicus** [comment]
Issue with prop target="_blank" with href set to a "blob" does not work with chrome 64.0.3282.140.

`<FileField src="rawFile.preview" title="rawFile.name" target="_blank"/>`

Works correctly with Firefox.

**Comment by djhi** [comment]
Do not comment on pull requests, please open an issue

**Pull request merged**